### PR TITLE
Enable registry-proxy for CNI users

### PR DESCRIPTION
**Pull request by Bregor**
PR to allow usage of https://github.com/deis/registry-proxy/pull/11

### CODE REVIEW

**Comment by bacongobbler** [comment]
Would you mind adding additional documentation so others know how to use this feature? Thanks! :)

**Comment by Bregor** [comment]
@bacongobbler you mean more comments in manifests or in some other place?

**Comment by bacongobbler** [comment]
I'm meaning in deis.com/docs/workflow. If you could add something in [Production Deployments](https://github.com/deis/workflow/blob/master/src/managing-workflow/production-deployments.md) on running Workflow with a Kubernetes cluster with CNI and how to configure this feature in there, that'd be fantastic.

**Comment by Bregor** [comment]
OK, sure. Will do it shortly.

**Comment by Bregor** [comment]
@bacongobbler should I add example right to https://github.com/deis/workflow/blob/master/src/managing-workflow/production-deployments.md, or make a reference to https://github.com/deis/distribution/blob/master/docs/configuration.md and fork those repo too?

File: charts/workflow/values.yaml [hunk 1]
```diff
@@ -57,6 +65,8 @@ global:
   # - true: deis-router will not be deployed. Workflow will not be usable until a Kubernetes ingress controller is installed.
   # - false: deis-router will be deployed (default).
   experimental_native_ingress: false
+  # If you're with CNI
```
Comment (vdice) [inline charts/workflow/values.yaml]: maybe rephrase this comment as `# If the Kubernetes cluster uses CNI`

**Comment by vdice** [comment]
@Bregor I think the first link (https://github.com/deis/workflow/blob/master/src/managing-workflow/production-deployments.md) which is here in the same repo, is fine for now.

**Comment by Bregor** [comment]
Docs added in https://github.com/deis/workflow/pull/801/files#diff-e00bc5e4152371ec0f45a782e3235b87
